libopenage/renderer/resources/texture_info.h

```diff
@@ -15,7 +15,8 @@ namespace renderer {
 namespace resources {
 
 /// How the pixels are represented in a texture.
-enum class pixel_format {
+enum class pixel_format
+{
```

Comment:
Why newline before {  😮

Reply:
Looks like a bug in clang-format causes the settings to be ignored: https://reviews.llvm.org/D106349